liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/CreateServerMojo.java

```diff
@@ -126,7 +126,10 @@ private void exportParametersToXml() throws Exception {
         configDocument.createElement("userDirectory", userDirectory);
         configDocument.createElement("serverOutputDirectory", new File(outputDirectory, serverName));
         configDocument.createElement("serverName", serverName);
-        
+
```

Comment:
you can delete 130 and 132 since configDocument.createElement() method has already handled null and exist.